.github/workflows/update-3rdparty.yml

```diff
@@ -141,10 +141,10 @@ jobs:
           submodules: true
           fetch-depth: 0
 
-      - name: update submodule to latest in 5.1 branch
+      - name: update submodule to latest in 5.1-syntax_5.1 (i.e. only valid for Lua 5.1 scripts) branch
```

Comment:
I'd keep this as-is, don't need to go into such detail at this level.

Reply:
I think we need to mention it *once* - so that it is clear why we are not using the `5.1` branch instead.

Reply:
Sure, and it is mentioned in other places already. Don't need to do it here as well.

Reply:
Where, precisely?

Reply:
Where, precisely?

Reply:
![image](https://user-images.githubusercontent.com/110988/156841798-0f0730ef-d72e-45fb-a06b-fe79367ea315.png)

Reply:
The problem with using the `name` for it is that it's not meant to be a long one:

![image](https://user-images.githubusercontent.com/110988/156841915-d542a566-3cba-4bc5-921a-a7ba70bedadc.png)

Reply:
Okay, somehow whatever was sticking for me on GitHub seems to have unglued itself.

So what is the problem? Is it that there are now four places where I have uses `5.1-syntax_5.1` instead of `5.1` - that you have pictured - because that is the branch that is being used - or the single case (the second of those four) where I have given a full explanation? I want to avoid saying the `5.1` branch because that is explicitly NOT the branch we want.

If we really must keep the texts short then I would drop most of the references to `5.1` except the `git checkout` one (obviously) but leave the long explanation one that will appear in the body of the auto generated PR/commit message.

Your thoughts, please?

Reply:
Okay, somehow whatever was sticking for me on GitHub seems to have unglued itself.

So what is the problem? Is it that there are now four places where I have uses `5.1-syntax_5.1` instead of `5.1` - that you have pictured - because that is the branch that is being used - or the single case (the second of those four) where I have given a full explanation? I want to avoid saying the `5.1` branch because that is explicitly NOT the branch we want.

If we really must keep the texts short then I would drop most of the references to `5.1` except the `git checkout` one (obviously) but leave the long explanation one that will appear in the body of the auto generated PR/commit message.

Your thoughts, please?

Reply:
Okay, somehow whatever was sticking for me on GitHub seems to have unglued itself.

So what is the problem? Is it that there are now four places where I have uses `5.1-syntax_5.1` instead of `5.1` - that you have pictured - because that is the branch that is being used - or the single case (the second of those four) where I have given a full explanation? I want to avoid saying the `5.1` branch because that is explicitly NOT the branch we want.

If we really must keep the texts short then I would drop most of the references to `5.1` except the `git checkout` one (obviously) but leave the long explanation one that will appear in the body of the auto generated PR/commit message.

Your thoughts, please?

Reply:
Okay, somehow whatever was sticking for me on GitHub seems to have unglued itself.

So what is the problem? Is it that there are now four places where I have uses `5.1-syntax_5.1` instead of `5.1` - that you have pictured - because that is the branch that is being used - or the single case (the second of those four) where I have given a full explanation? I want to avoid saying the `5.1` branch because that is explicitly NOT the branch we want.

If we really must keep the texts short then I would drop most of the references to `5.1` except the `git checkout` one (obviously) but leave the long explanation one that will appear in the body of the auto generated PR/commit message.

Your thoughts, please?

Reply:
Can you delete those duplicates - I can't as the drop down menu doesn't appear any more (like previews or the posted messages neither :roll_eyes:)

Reply:
The `- name:` is not what will be in the body of the PR, you are across that right?